### PR TITLE
nim: 0.20.0 -> 0.20.2

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

maintainers: @ehmry @peterhoeg

```
/nix/store/v2wx8k0zcg47paiac60v950lpwzcr8g3-nim-0.20.0	 233.7M
/nix/store/8vyg93hwqmadzr5l327c5sjrrl6n275w-nim-0.20.2	 233.9M
```

```
pbb@uwu ~/nixpkgs> nix-shell -p nix-review --run "nix-review wip"
$ git fetch --force https://github.com/NixOS/nixpkgs master:refs/nix-review/0
$ git worktree add /home/pbb/.cache/nix-review/rev-fcd940feb1b313da7f8b912214241a87c3512bde-dirty/nixpkgs f7579996c712b52aca134beb53f6a333f91a429f
Preparing worktree (detached HEAD f7579996c71)
Checking out files: 100% (19451/19451), done.
HEAD is now at f7579996c71 Merge pull request #62566 from r-ryantm/auto-update/serviio
$ nix-env -f /home/pbb/.cache/nix-review/rev-fcd940feb1b313da7f8b912214241a87c3512bde-dirty/nixpkgs -qaP --xml --out-path --show-trace
Applying `nixpkgs` diff...
$ nix-env -f /home/pbb/.cache/nix-review/rev-fcd940feb1b313da7f8b912214241a87c3512bde-dirty/nixpkgs -qaP --xml --out-path --show-trace --meta
$ nix build --no-link --keep-going --max-jobs 8 --option build-use-sandbox true -f /home/pbb/.cache/nix-review/rev-fcd940feb1b313da7f8b912214241a87c3512bde-dirty/build.nix
[3 built, 7 copied (7.1 MiB), 2.7 MiB DL]
3 package were build:
mosdepth nim nrpl

[0.0 MiB DL]
$ nix-shell /home/pbb/.cache/nix-review/rev-fcd940feb1b313da7f8b912214241a87c3512bde-dirty/shell.nix
[nix-shell:~/.cache/nix-review/rev-fcd940feb1b313da7f8b912214241a87c3512bde-dirty]$ exit
$ git worktree prune
pbb@uwu ~/nixpkgs> 
```

### CODE REVIEW

**Comment by mmahut** [comment]
@GrahamcOfBorg build nim

File: pkgs/development/compilers/nim/default.nix [hunk 1]
```diff
@@ -105,7 +105,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Statically typed, imperative programming language";
-    homepage = https://nim-lang.org/;
+    homepage = "https://nim-lang.org/";
```
Comment (peterhoeg) [inline pkgs/development/compilers/nim/default.nix]: Any particular reason for turning this into a string?

Comment (mmahut) [reply pkgs/development/compilers/nim/default.nix]: @peterhoeg https://github.com/NixOS/rfcs/pull/45

Comment (peterhoeg) [reply pkgs/development/compilers/nim/default.nix]: I wasn't aware of that - thanks!

**Pull request merged**